src/test/java/teammates/test/cases/util/ActivityLogEntryTest.java

```diff
@@ -41,11 +41,11 @@ public void testActivityLogEntryClass() {
         appLog.setTimeUsec(0);
         entry = new ActivityLogEntry(appLog);
         
-        assertEquals("<a href=\"URL?user=UserId\" class=\"text-success bold\" target=\"_blank\">instructorHome</a>",
-                     entry.getActionInfo());
-        assertEquals("01-01-1970 07:30:00", entry.getDateInfo());
-        assertEquals("Message", entry.getMessageInfo());
-        assertEquals("UserId", entry.getPersonInfo());
+        assertEquals("instructorHome",
+                     entry.getServletName());
```

Comment:
This line is short enough to be included in the same line with the previous one.